src/encoded/audit/experiment.py

```diff
@@ -262,11 +262,14 @@ def audit_experiment_missing_unfiltered_bams(value, system, files_structure):
 def audit_experiment_with_uploading_files(value, system, files_structure):
     if files_structure.get('original_files'):
         for file_object in files_structure.get('original_files').values():
-            if file_object['status'] in ['uploading', 'upload failed', 'content error']:
-                detail = 'Experiment {} '.format(value['@id']) + \
-                    'contains a file {} '.format(file_object['@id']) + \
-                    'with the status {}.'.format(file_object['status'])
+            detail = ('Experiment {} contains a file {} '
```

Comment:
Nice, but let's flip it around so it only builds the detail with we have an AuditFailure
```
        for file_object in files_structure.get('original_files').values():
            category = None
            if file_object['status'] in ['upload failed', 'content error']:
                category = 'file validation error'
            elif file_object['status'] == 'uploading':
                category = 'file in uploading state'
            if category:
                detail = ('Experiment {} contains a file {} '
                          'with the status {}.'.format(value['@id'],
                                                       file_object['@id'],
                                                       file_object['status']))
                yield AuditFailure(category, detail, level='INTERNAL_ACTION')
```

Other than this I think this is ready to go.